lib/resources/r4/practitioner.yaml

```diff
@@ -24,3 +26,112 @@ fields:
       - display: v2 Identifier Type
         system: http://terminology.hl7.org/CodeSystem/v2-0203
         info_link: https://hl7.org/fhir/r4/v2/0203/index.html
+
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  action: create
+  example: |
+    {
+      "resourceType": "Practitioner"
+    }
+  note: resourceType must be Practitioner.
+  url: http://hl7.org/fhir/R4/json.html#resources
+
+- name: name
+  required: 'Yes'
+  cardinality: 1..1
+  type: HumanName
+  description: The name associated with the practitioner.
+  action: create
+  example: |
+    {
+      "name": [
+        {
+          "family": "Williams",
+          "given": [
+            "Rory", "James McAfee"
+          ],
+          "prefix": [
+            "Dr."
+          ],
+          "suffix": [
+            "M.D."
+          ],
+          "period": {
+            "start": "2019-12-01T00:00:00.000Z"
+          }
+        }
+      ]
+    }
+  note: |
+    <ul>
+      <li> Exactly one name <b>must</b> be provided. The provided name must meet the following conditions:
+        <ul>
+          <li> Must include both `given` and `family`. </li>
+          <li> Up to two `given` names may be provided. If additional given names are required they must be provided in the second value as a single string separated by blank spaces. </li>
```

Comment:
maybe 'Up to two values may be present in the `given` names array. If more than two given names are required the second value in the array accepts whitespace-delimited strings.' ?

does the number of whitespaces matter?

Reply:
> does the number of whitespaces matter?

Each value in that array is limited to 100 characters, so number of whitespaces matters from a functional perspective **there**. A Practitioner could be written with a second given name like `John <insert 80 spaces> Ryan` and we wouldn't fail or anything

Reply:
https://github.com/cerner/fhir.cerner.com/pull/330/commits/7a2bf1e0a6700f184afdeff837657a6d58827de5#diff-c593d32e3c230e494be00194f16e6986R74